defold-spine/src/gui_node_spine.cpp

```diff
@@ -500,13 +501,61 @@ static void UpdateBones(InternalGuiNode* node)
     dmVMath::Vector4 scale = dmGui::GetNodeProperty(scene, node->m_GuiNode, dmGui::PROPERTY_SCALE);
     for (uint32_t i = 0; i < num_bones; ++i)
     {
-        spBone* bone = node->m_Bones[i];
         dmGui::HNode gui_bone = node->m_BonesNodes[i];
-
+        spBone* bone = node->m_Bones[i];
         UpdateTransform(scene, gui_bone, bone);
     }
 }
 
+static void FindGuiBones(InternalGuiNode* node, dmGui::HScene scene, dmGui::HNode hnode)
+{
+    // We assume the order is the same as when we created them in the original spine node
+    node->m_BonesNodes.Push(hnode);
+
+    dmGui::HNode child = dmGui::GetFirstChildNode(scene, hnode);
+
+    while (child != dmGui::INVALID_HANDLE)
+    {
+        if (dmGui::GetNodeIsBone(scene, child)) // We cannot have bones as a child of another node type
+        {
+            FindGuiBones(node, scene, child);
+        } else {
```

Comment:
Should we have an empty "else" here?

Reply:
Ah, old stuff where I had my debug print :)